src/FunctionalExtras/Data/ValueTupleExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
I think this file is misnamed.  It's dealing with `System.Tuple` instances, not `System.ValueTuple` instances.

Reply:
I think it didn't switch context when I clicked… = \

Reply:
Fixed in [783e8ab](https://github.com/jlmorgan/csharp-functional-extras/pull/11/commits/783e8ab2605bb4e45e3098d36592f6c6b90d819b).